concepts/while-and-repeat-loops/about.md

```diff
@@ -0,0 +1,71 @@
+# About
+
+A [while loop][while-loops] is a control flow statement that allows code to be executed while a certain condition is true.
+Swift provides two kinds of while loops: while and repeat-while loops.
+While loop's condition is evaluated before the loop body is executed; if the condition is false, the loop body will not be executed at all.
+while-loops in Swift have the following structure:
```

Comment:
One sentence starts with "While loop's" and the next with "while-loops". It would be good to make these consistent.